src/utils/contracts.ts

```diff
@@ -48,6 +48,22 @@ export const CXIP_ERC721_ADDRESSES: {[key in Environment]: string} = {
   [Environment.mainnet]: '0x690f4b7e0a102047d442CA3FEcDbB024Cb6b1FC5',
 }
 
+export const METADATA_RENDERER_ADDRESS: {[key in Environment]: string} = {
+  [Environment.localhost]: '0x0000000000000000000000000000000000000000',
+  [Environment.experimental]: '0x0000000000000000000000000000000000000000',
+  [Environment.develop]: '0x6420ffC8390506BBC9d894f54b948Be2BE40128d',
+  [Environment.testnet]: '0x0000000000000000000000000000000000000000',
+  [Environment.mainnet]: '0x0000000000000000000000000000000000000000',
+}
+
+export const HOLOGRAPH_FACTORY_PROXY_ADDRESS: {[key in Environment]: string} = {
```

Comment:
@alexanderattar @ACC01ADE is this really needed? I thought we derive this address from the main `HOLOGAPH_ADDRESS`.

Reply:
done